README.md

```diff
@@ -12,7 +12,7 @@ urlFragment: ms-identity-python-webapp
 ## About this sample
 
 > This sample is also available as a quickstart for the Microsoft identity platform:
-[Quickstart: Add sign-in with Microsoft to a Python web app]("https://docs.microsoft.com/azure/active-directory/develop/quickstart-v2-python-webapp")
+[Quickstart: Add sign-in with Microsoft to a Python web app](https://docs.microsoft.com/en-us/azure/active-directory/develop/web-app-quickstart?pivots=devlang-python)

```

Comment:
Good catch! [My bad](https://github.com/Azure-Samples/ms-identity-python-webapp/pull/9/files). I was misled by the previous placeholder which contained that undesirable double quotation marks.

The proper fix, however, can probably remove the language specifier. I believed I was told - but I am not entirely sure now - that a language-less url will be automatically redirected into the local language based on a reader's language preference setting in browser.

Please try this out, and if it also works for you, you can accept this suggestion by one-click on github.

```suggestion
[Quickstart: Add sign-in with Microsoft to a Python web app](https://docs.microsoft.com/azure/active-directory/develop/web-app-quickstart?pivots=devlang-python)

```

Reply:
Thank you @rayluo. Removing the language specifier in the URL works, committing your suggestion now.